.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Thank you for contributing a pull request! Here are a few tips to help you:
+
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
+
+#### Summary
+<!--
+A description of what this pull request does.
+-->
+
+#### Ticket Link
+<!--
+If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.
+
+  Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX
+
+Otherwise, link the Jira ticket.
+-->
+
+#### Checklist
+<!--
+Please review each item and check the ones applicable for your pull request
+-->
+- [ ] Tested related changes <!-- See https://developers.mattermost.com/contribute/server/developer-workflow/#running-only-specific-server-unit-tests how to run specific unit tests. -->
+- [ ] Added or updated unit tests <!-- This should be done for every change, if possible -->
```

Comment:
Do we need the parenthetical `This should be done for every change, if possible `? It seems like having shortened the list of required checkboxes, we should expect these to be done and no longer have to say that we expect them.

Also, what about shortening `Added or updated` to just:
```suggestion
- [ ] Updated unit tests
```

Reply:
I see your point. But at the end, what is the whole point of this check list item anyway? Like, what does it change for the reviewers if this is checked or not? They will request unit test anyway if then think this is required.